mat64/dense_test.go

```diff
@@ -501,7 +501,7 @@ func (s *S) TestMul(c *check.C) {
 
 		// These probably warrant a better check and failure. They should never happen in the wild though.
 		temp.mat.Data = nil
-		c.Check(func() { temp.Mul(a, b) }, check.PanicMatches, "cblas: index of c out of range", check.Commentf("Test %d"))
+		c.Check(func() { temp.Mul(a, b) }, check.PanicMatches, "general: insufficient length", check.Commentf("Test %d"))
```

Comment:
Should this panic message be "blas: index out of range" to match https://github.com/gonum/blas/pull/66/files